web_responsive/README.rst

```diff
@@ -156,6 +156,7 @@ Contributors
 * Dennis Sluijk <d.sluijk@onestein.nl>
 * Sergio Teruel <sergio.teruel@tecnativa.com>
 * Alexandre Díaz <dev@redneboa.es>
+* Mathias Markl <mathias.markl@mukit.at>
```

Comment:
Unless you put this on `readme/CONTRIBUTORS.rst`, it will be lost on README regeneration.

Reply:
https://github.com/OCA/web/commit/2e911c6b028ecbd700ae041e7f426e4ca30703c1